boards/riscv/beagle_v_starlight/beagle_v_starlight.dts

```diff
@@ -0,0 +1,21 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+
+/dts-v1/;
+
+#include "starfive/starfive_jh7100_beagle_v.dtsi"
```

Comment:
as per @pdp7's recommendation, I would change to `beaglev_starlight_jh7100`

Also, please change to the recommended compat strings

https://github.com/beagleboard/beaglev-starlight#guidance-on-naming-conventions

Reply:
Fixed.